src/main/java/org/jenkins/plugins/lockableresources/BackwardCompatibility.java

```diff
@@ -46,6 +46,8 @@ public static void compatibilityMigration() {
         }
         queuedContexts.clear();
       }
+
+      resource.repairLabels();
```

Comment:
This is not required, this should be migrated silently by `readResolve`

Reply:
Great. In that case can be the function repairable private